espnet/bin/asr_train.py

```diff
@@ -596,6 +596,8 @@ def main(cmd_args):
         char_list = [entry.decode("utf-8").split(" ")[0] for entry in dictionary]
         char_list.insert(0, "<blank>")
         char_list.append("<eos>")
+        if args.dmode == "NAR":
+            char_list.append("<mask>")
```

Comment:
please add an explanation about `<mask>`

Reply:
This is probably problematic because RNN models do not have `dmode` in args.